test/imported/test_indexing.py

```diff
@@ -1085,15 +1085,15 @@ def test_index_src_datatype(self):
   def test_int_indices2d(self):
     # From the NumPy indexing example
     x = Tensor.arange(0, 12).reshape(4, 3)
-    rows = Tensor([[0, 0], [3, 3]])
-    columns = Tensor([[0, 2], [0, 2]])
+    rows = Tensor([[0, 0], [3, 3]], dtype=dtypes.int32)
```

Comment:
this looks like a bug in core tinygrad that we default convert dtype to float. i will check this tomorrow